docs/docs-content/integrations/rook-ceph.md

```diff
@@ -11,58 +11,336 @@ logoUrl:
+             owner: 0
+             group: 0
+             content: |
+               rbd
+               ceph
+             encoding: ""
+             ownerstring: ""
+     after-upgrade:
+       - name: "Erase Old Partitions on Boot Disk"
+         commands:
+           - wipefs -a /dev/sdb
+   ```
 
-**1.8.3**
+2. Click on the Rook-Ceph layer. In the upper-right corner of the Rook-Ceph layer's YAML editing interface, click
+   **Presets**. Set the preset to either single-node or multi-node depending on your cluster configuration.
+
+3. If you chose the **Single Node Cluster** preset, skip this step.
+
+   If you chose the **Multi Node Cluster with Replicas** preset, set the value of
+   `manifests.storageClass.volumeBindingMode` to `Immediate`.
+
+### Access Ceph Dashboard
+
+The Ceph dashboard gives you an overview of the status of your Ceph cluster, including overall health, and the status of
+all Ceph daemons. By default, the Dashboard is exposed as a ClusterIP-type service on the port 7000 on single node
+clusters.
+
+1. Issue the following command to view the service and find its cluster IP and port.
+
+   ```shell
+   kukubectl --namespace rook-ceph get svc | grep dashboard
+   ```
+
+   ```hideClipboard
+   rook-ceph-mgr-dashboard   ClusterIP   192.169.32.142   <none>        7000/TCP            15m
+   ```
+
+2. If you are on a node of the cluster, you can visit the dashboard by visiting the cluster IP and the exposed port.
+
+   If you are remotely accessing the cluster, you can issue the following command to enable port forwarding from your
+   local machine to the dashboard service.
+
+   ```shell
+   kukubectl port-forward svc/rook-ceph-mgr-dashboard -n rook-ceph 8443:7000 &
+   ```
+
+   If your dashboard service is exposed on a different port, replace 7000 with the port that the dashboard service is
+   exposed on.
+
+3. Once you can connect to the dashboard, you need to provide the login credentials to access the dashboard. Rook
+   creates a default user named `admin` and generates a secret called `rook-ceph-dashboard-password` in the namespace of
+   the Rook-Ceph cluster. To retrieve the generated password, issue the following command:
+
+   ```shell
+   kubectl -n rook-ceph get secret rook-ceph-dashboard-password -o jsonpath="{['data']['password']}" | base64 --decode && echo
+   ```
+
+4. Use the password you receive in the output with the username `admin` to log in to the Ceph Dashboard.
 
 </TabItem>
 
-<TabItem label="1.5.x" value="1.5.x">
+<TabItem label="Deprecated" value="Deprecated">
 
-**1.5.9**
+Older versions of this pack is deprecated. Please use the later versions to take advantage of new features and
+improvements.
 
 </TabItem>
 
 </Tabs>
 
+## Terraform
+
+```tf
+
+data "spectrocloud_registry" "registry" {
+  name = "Public Repo"
+}
+
+data "spectrocloud_pack_simple" "pack" {
+  name    = "csi-rook-ceph-addon"
+  version = "1.12.7"
+  type = "helm"
+  registry_uid = data.spectrocloud_registry.registry.id
+}
+
```

Comment:
```suggestion
```tf
data "spectrocloud_registry" "registry" {
  name = "Public Repo"
}

data "spectrocloud_pack_simple" "pack" {
  name    = "csi-rook-ceph-addon"
  version = "1.12.7"
  type = "helm"
  registry_uid = data.spectrocloud_registry.registry.id
}
```

Reply:
Just removing the extra spaces around code here